### ISSUE TITLE
TextInput ScrollEnabled causes unrecognized selector crash

### ISSUE THREAD

**Issue by L-Yeiser** [facebook/react-native#21339]
- [ ] Review the documentation: https://facebook.github.io/react-native
  - [ ] Search for existing issues: https://github.com/facebook/react-native/issues
  - [ ] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
```

## Description
Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know.

## Reproducible Demo
Let us know how to reproduce the issue. Include a code sample, share a project, or     share an app that reproduces the issue using https://snack.expo.io/. Please follow the guidelines for providing a MCVE: https://stackoverflow.com/help/mcve

**Comment by ggtmtmgg** [comment]
@L-Yeiser 
`<TextInput scrollEnabled={false} />` only works with `multiline={true}`.

https://facebook.github.io/react-native/docs/textinput#scrollenabled

**Comment by L-Yeiser** [comment]
@ggtmtmgg you are right. I should have been clearer in my issue description and updated the repo to be more explanatory. Passing the `scrollEnabled` prop without `multiline` causes the above error - even when the value is `scrollEnabled={true}`. When docs say that `scrollEnabled=false` does not work without `multiline=true` my assumption is that the property will not work (aka it is not a multiline text there is no scroll to disable) not that the app will crash. We have a re-usable `TextInput` component with a handful of default props. It is easy enough to add protection to not pass  the `scrollEnabled` prop without the `multiline` prop but it does feel like a bug that passing one property without the other causes a fetal error. I apologize for not being clearer in the initial issue description.
